internal/controllers/vehicle_subscription_controller.go

```diff
@@ -231,8 +300,104 @@ func (v *VehicleSubscriptionController) SubscribeAllVehiclesToWebhook(c *fiber.C
 		}
 		successCount++
 	}
+	return c.Status(http.StatusCreated).JSON(fiber.Map{
+		"message": fmt.Sprintf("Successfully subscribed %d vehicles to the webhook", successCount),
+	})
+}
+
+// SubscribeMultipleVehiclesToWebhook godoc
+// @Summary      Subscribe multiple vehicles to a webhook
+// @Description  Accepts a comma-separated list of vehicle token IDs and subscribes each one to the specified webhook event.
+// @Tags         Vehicle Subscriptions
+// @Produce      json
+// @Param        eventID path string true "Event ID"
+// @Param        vehicleTokenIDs query string true "Comma-separated list of vehicle token IDs"
+// @Param        Authorization header string true "Bearer token"
+// @Success      201 {object} map[string]string "Successfully subscribed X vehicles to the webhook"
+// @Failure      400 {object} map[string]string "Event ID and vehicle token IDs are required"
+// @Failure      401 {object} map[string]string "Unauthorized"
+// @Failure      500 {object} map[string]string "Internal server error"
+// @Router       /subscriptions/{eventID} [post]
+func (v *VehicleSubscriptionController) SubscribeMultipleVehiclesToWebhook(c *fiber.Ctx) error {
+	eventID := c.Params("eventID")
+	if eventID == "" {
+		return c.Status(http.StatusBadRequest).JSON(fiber.Map{"error": "Event ID is required"})
+	}
+
+	vehicleIDsParam := c.Query("vehicleTokenIDs")
+	if vehicleIDsParam == "" {
+		return c.Status(http.StatusBadRequest).JSON(fiber.Map{"error": "Vehicle token IDs are required"})
+	}
+
+	ids := strings.Split(vehicleIDsParam, ",")
 
+	devLicense, err := getDevLicense(c, v.logger)
+	if err != nil {
+		return c.Status(http.StatusUnauthorized).JSON(fiber.Map{"error": "Unauthorized"})
+	}
+
+	var successCount int
+	for _, idStr := range ids {
+		idStr = strings.TrimSpace(idStr)
+		tokenDecimal := types.Decimal{}
```

Comment:
one thing we realized, is dealing with numeric(78,0) in the database is a pain and we have accepted just to use `bigint` since it can still go up to `9,223,372,036,854,775,807`. 
If it is helpful, feel free to migrate your database to just use bigint to represent token ID's.